cmd/manager/main.go

```diff
@@ -713,6 +715,11 @@ func startOperator(ctx context.Context) error {
 		return err
 	}
 
+	isOpenshift, err := isOpenShift(clientset)
```

Comment:
Move this further up above L691 and use it as an input for `determineSetDefaultSecurityContext` so we don't have to run the detection twice.

Reply:
changed here ab208bebc3bfb38a32ed55019eabd3f79c8c3a11